lib/memorial.js

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from "react";
+
+/*
+Use mock data for now
+ */
+export function useMemorial(memorialID) {
+  const [loading, setLoading] = useState(false);
+  const [data, setData] = useState(undefined);
+  const [error] = useState("");
+
+  useEffect(() => {
+    // fetch id if it exists
+    if (memorialID) {
+      setLoading(true);
+      setTimeout(() => {
+        setData(MOCK_DATA);
+        setLoading(false);
+      }, 500);
+    } else {
+      // set it to undefined if no id
+      setData(undefined);
+    }
+  }, [memorialID]);
+
+  return { memorial: data?.memorial, role: data?.memorial, loading, error };
+}
+
+const MOCK_DATA = {
```

Comment:
Great Idea. To simulate a memorial for testing purposes. This also helps because there is a clarity of what the UI and CSS will look like